src/datomic_rest_api/rest/fields/gene.clj

```diff
@@ -1451,10 +1477,10 @@
                 [?tdiv :db/ident ?div]]
        db (:db/id gene))
     (map
-     (fn [[div term code anno]]
+     (fn [[div term anno]]
        {:division div
-        :term     (entity db term)
-        :anno     (entity db anno)}))
+        :term (entity db term)
```

Comment:
I'd probably stick to using the `d` namespace for the datomic api here.
e.g `(d/entity db term)`
(minor nit-pick)